processing/src/test/java/io/druid/query/extraction/LookupDelegatorTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.extraction;
+
+import com.fasterxml.jackson.databind.InjectableValues;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.druid.jackson.DefaultObjectMapper;
+import org.easymock.EasyMock;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+
+public class LookupDelegatorTest
+{
+  private final LookupReferencesManager lookupReferencesManager = EasyMock.createMock(LookupReferencesManager.class);
+  private final ObjectMapper mapper = new DefaultObjectMapper();
+  private LookupExtractor lookupExtractor;
+  private final LookupExtractorFactory mockLookupFactory = EasyMock.createMock(LookupExtractorFactory.class);
+  private final LookupExtractor mockLookup = EasyMock.createMock(LookupExtractor.class);
+  private final String name = "name";
+
+  @Test
+  public void testSerDeser() throws IOException
```

Comment:
Having tests before the `setUp` and `tearDown` is a little confusing.  Usually nice to have all the tests in the same section.